5.1.i_have_no_vinger.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+import random
+
+fDate = datetime.datetime.strptime(input("write a date (YYYY-MM-DD):"), '%Y-%m-%d')
+sDate = datetime.datetime.strptime(input("write a date (YYYY-MM-DD):"), '%Y-%m-%d')
+
+randDate = datetime.datetime.fromtimestamp(random.uniform(fDate.timestamp(), sDate.timestamp()))
+
+if (randDate.weekday() == 0):
```

Comment:
Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.